sabre1041/managing-ocp-install-beyond
# adding the initial commit of the alert manager apb

To test this (all from master host):

- ```sudo -i```
- clone the branch on the master host
- ```yum -y install apb```
- give rights to the <student-name> user: ```oc adm policy add-cluster-role-to-user cluster-admin <student-name>```
- login as user: ```oc login -u <student-name>```
- confirm with ```oc whoami```
- change to the openshift-metrics project:  ```oc project openshift-metrics```
- cd into the alert manager directory: ```cd /root/managing-ocp-install-beyond/alert-manager-apb```
- build the apb: ```apb build```
- push the apb to the service catalog: ```apb push```
- launch the application from web UI
- - log into openshift web console
- - change to the openshift-metrics project
- - launch the alert application
- go back to CLI, in the openshift-metrics project, ```oc get all``` confirm there is an imagestream, buildconfig, pods, etc... look for errors

The password in the PR is set to password, you'll want to change that to our lab password.
